xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemux.cpp

```diff
@@ -45,8 +64,48 @@ std::string CDemuxStreamAudio::GetStreamType()
       strInfo = "TrueHD ";
       break;
     case AV_CODEC_ID_AAC:
-      strInfo = "AAC ";
+    {
+      switch (profile)
+      {
+        case FF_PROFILE_AAC_LOW:
+        case FF_PROFILE_MPEG2_AAC_LOW:
+          strInfo = "AAC-LC ";
+          break;
+        case FF_PROFILE_AAC_HE:
+        case FF_PROFILE_MPEG2_AAC_HE:
+          strInfo = "HE-AAC ";
+          break;
+        case FF_PROFILE_AAC_HE_V2:
+          strInfo = "HE-AACv2 ";
+          break;
+        case FF_PROFILE_AAC_SSR:
+          strInfo = "AAC-SSR ";
+          break;
+        case FF_PROFILE_AAC_LTP:
+          strInfo = "AAC-LTP ";
+          break;
+        default:
+        {
+          // Try check by ISO codec full string
```

Comment:
Is it maybe worth adding that DashIF url here as a reference point for the codec strings?

Reply:
comment changed however its RFC 6381